### ISSUE TITLE
Testnet failed consensus, one node is stucked

### ISSUE THREAD

**Issue by spikeekips** [bosnet/sebak#700]
Our previous testnet was stuck
```
CONSENSUS 101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
CONSENSUS 101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
CONSENSUS 101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
CONSENSUS 101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
CONSENSUS 101921 BNtQLiiCjhfyDkfcCCgpSQGXegNFkHuvX4JEk3Le7USm
BOOTING   101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
BOOTING   101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
BOOTING   101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
BOOTING   101929 98scSYZjEjDt4V34yBtpuqqNqX3A9RnWutgQjZ9o7Xh9
SYNC       82500 CfLMLCSXn3ZiWhoqYKaM4M61eRHoSehPz5hGSsXzgEUN
```
The last one, `CfLM...gEUN` was behind.

**Comment by spikeekips** [comment]
The detailed logs you can find at https://drive.google.com/drive/folders/1PeJ2rOC444IzMm9eS5m1bL1Ly57137QO?usp=sharing .

**Comment by anarcher** [comment]
```
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
{
  "hash": "CfLMLCSXn3ZiWhoqYKaM4M61eRHoSehPz5hGSsXzgEUN",
  "height": 82500,
  "tx_root": "HvXQzJHbg6WTYewkKE4kQ7A2y453hn4fQnUugHWvjkTB",
  "proposer": "GDTEPFWEITKFHSUO44NQABY2XHRBBH2UBVGJ2ZJPDREIOL2F6RAEBJE4",
  "round": 0
}
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
{
  "hash": "ZqH82rEtHwPUEYGfZwz1U7NHHKQg3nWsF25Ro18AUTe",
  "height": 82500,
  "tx_root": "GTWvdpUnvATYmVsm6crKB9h7WQEM4D5AsWJxFd7ZxNTV",
  "proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
  "round": 1
}
```

```
"proposer": "GAYGELM74WJMKSLDN5YP2VAMP64WC4IXIGICUNK2SCVIT7KPTLY7M3MW",
 "proposer": "GDTEPFWEITKFHSUO44NQABY2XHRBBH2UBVGJ2ZJPDREIOL2F6RAEBJE4",
```

**Comment by Charleslee522** [comment]
fixed in #711
